usage/src/main/java/org/killbill/billing/usage/api/BaseUserApi.java

```diff
@@ -51,16 +52,16 @@ public BaseUserApi(final OSGIServiceRegistration<UsagePluginApi> pluginRegistry)
     //   and the usage module will look for data inside its own table.
     // * If not, a possibly empty (or not) list should be returned (and the usage module will *not* look for data inside its own table)
     //
-    protected List<RawUsageRecord> getAccountUsageFromPlugin(final LocalDate startDate, final LocalDate endDate, final TenantContext tenantContext) {
-        return getUsageFromPlugin(null, startDate, endDate, tenantContext);
+    protected List<RawUsageRecord> getAccountUsageFromPlugin(final LocalDate startDate, final LocalDate endDate, final UsageContext usageContext) {
+        return getUsageFromPlugin(null, startDate, endDate, usageContext);
     }
 
-    protected List<RawUsageRecord> getSubscriptionUsageFromPlugin(final UUID subscriptionId, final LocalDate startDate, final LocalDate endDate, final TenantContext tenantContext) {
-        return getUsageFromPlugin(subscriptionId, startDate, endDate, tenantContext);
+    protected List<RawUsageRecord> getSubscriptionUsageFromPlugin(final UUID subscriptionId, final LocalDate startDate, final LocalDate endDate, final UsageContext usageContext) {
+        return getUsageFromPlugin(subscriptionId, startDate, endDate, usageContext);
     }
 
-    private List<RawUsageRecord> getUsageFromPlugin(@Nullable final UUID subscriptionId, final LocalDate startDate, final LocalDate endDate, final TenantContext tenantContext) {
-        Preconditions.checkNotNull(tenantContext.getAccountId(), "TenantContext has no accountId");
+    private List<RawUsageRecord> getUsageFromPlugin(@Nullable final UUID subscriptionId, final LocalDate startDate, final LocalDate endDate, final UsageContext usageContext) {
+        Preconditions.checkNotNull(usageContext.getAccountId(), "TenantContext has no accountId");
```

Comment:
Nit: fix comment (`UsageContext`).

Reply:
Fixed in https://github.com/killbill/killbill/pull/1648/commits/eaae451610ed4d3fbe6cafc9dfdf65e76de4cd5d